libc/src/sys/ioctl/ioctl.h

```diff
@@ -0,0 +1,20 @@
+//===-- Implementation header for mmap function -----------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_SYS_IOCTL_IOCTL_H
+#define LLVM_LIBC_SRC_SYS_IOCTL_IOCTL_H
+
+#include <sys/mman.h> // For size_t and off_t
```

Comment:
We prefer internal type definitions now. Double check with @nickdesaulniers.

Reply:
I dont really get it, can you elaborate what are you referring to?

Reply:
@michaelrj-google and I are still discussing this preference and will revisit it on Wednesday. For now the code is fine.

What @SchrodingerZhu was referring to was including `include/llvm-libc-types/off_t.h` and `include/llvm-libc-types/size_t.h` for these declarations instead.

Reply:
That said, `size_t` and `off_t` do not appear in the signature of this function, so this include is not necessary in this header.

In general we prefer direct includes rather than transitive includes.

Please remove this include line here.

Reply:
removed the include